lib/online_order.rb

```diff
@@ -0,0 +1,79 @@
+require 'pry'
+require 'csv'
+require_relative './order.rb'
+require_relative './customer.rb'
+
+module Grocery
+
+  class OnlineOrder < Grocery::Order
+    attr_reader :id, :customer_id, :customer
+    attr_accessor :fill_status, :prducts
+
+    ONLINE_ORDERS = "support/online_orders.csv"
+
+    def initialize(order_id, products, cust_id, fill_status)
+      @id = order_id.to_i
+      @products = products
+      @customer_id = cust_id.to_i
+      @fill_status = fill_status.to_sym
+      @customer = Grocery::Customer.find(@customer_id)
+    end
+
+    def total
+      unless super() == 0
+        total = super() + 10.00
+        total = total.round(2)
+        return total
+      else
+        return total = 0.00
```

Comment:
On line 28, you don't need to set the local variable `total`, since as soon as you return from this method it's gone.